index.html

```diff
@@ -275,10 +275,17 @@ <h3>
                 <li>Test Suites</li>
               </ul>
             </li>
+            <li>Other VC Data Integrity proof types:
+              <ul>
+                <li>VC Data Integrity - <a href="https://or13.github.io/lds-pgp2021"></a>PGP</li>
+                <li>VC Data Integrity - <a href="https://ns.did.ai/suites/multikey-2021"></a>Multbase / MultiCodec</li>
```

Comment:
This is the only line I would object to. We put `publicKeyMultibase` and Multikey in DID Core with the explicit goal to finish standardizing it in the VC 2.0 WG. This is a normative deliverable, to be defined in Data Integrity, and must remain as such.